api/app/controllers/mno_enterprise/jpi/v1/admin/invites_controller.rb

```diff
@@ -39,7 +39,7 @@ def send_org_invite(invite)
       # Generate token if not generated
       user.send(:generate_confirmation_token!) if !user.confirmed? && user.confirmation_token.blank?
 
-      MnoEnterprise::SystemNotificationMailer.organization_invite(invite).deliver_later
+      MnoEnterprise::SystemNotificationMailer.organization_invite(invite).deliver_now
```

Comment:
Why remove the `deliver_later`? By default it'll be performed now but if we add sidekiq it'll automatically use it.